### ISSUE TITLE
Performing Actions in Gmail Causing Browser to Become Disconnected from the Internet

### ISSUE THREAD

**Issue by feerlessleadr** [brave/brave-browser#21884]
## Description 
As the title states, I noted starting yesterday that anytime I would do anything in Gmail (send an email, look in drafts folder, schedule an email to send, etc.), after a period of maybe 30 seconds to 5 minutes, the internet for just Brave would no longer work. I confirmed that the internet is working on my laptop, as I have to use Edge for work, and while the internet was not working in Brave (tried many sites), it continued to work in Edge (using the same sites). 

The only way to fix this was to clear my browsing data ("Cookies and other side data" & "Cached images and files") and then restart brave. After the restart, the internet would begin working in Brave once again. This same issue is happening on my work laptop, my personal surface, as well as my Windows server (all running Windows 10 Pro). 

## Steps to Reproduce

   1. Open gmail
   2. Compose new message with any sender/text 
   3. Select 'Schedule Send' next to send button
   4. Wait between 30 seconds - 2 minutes, no websites will load in Brave

## Actual result:
No websites work in Brave, as if I am not connected to the internet

## Expected result:
Websites should work normally

## Reproduces how often: 
Easily reproduced

## Brave version (brave://version info)

<html>
<body>

Brave | 1.36.119 Chromium: 99.0.4844.83 (Official Build) (64-bit)
-- | --
Revision | b11086e62d7c1a44b0942ac5568d22a425c7ae35-refs/branch-heads/4844_74@{#5}
OS | Windows 10 Version 20H2 (Build 19042.1586)

</body>
</html>

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? I do not know, I do not have that installed
- Can you reproduce this issue with the nightly channel? I do not know, I do not have that installed

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields?  No
- Does the issue resolve itself when disabling Brave Rewards? No
- Is the issue reproducible on the latest version of Chrome? I do not have Chrome installed

## Miscellaneous Information:

None

**Comment by rebron** [comment]
cc: @ryanbr @Brave-Matt 

@feerlessleadr Going to Gmail prevents you from visiting other websites thereafter? When you open up a new tab does that prevent you from viewing websites or are we talking about the same tab. 

Might not be it but can you check brave://settings/socialBlocking and toggle Allow Google logins to see if that might be causing the error. I'm not sure about the list of web sites you're trying to access.

**Comment by feerlessleadr** [comment]
Thanks for responding so quickly. It is both in the same tab, as well as in new tabs (it also persists after I close and reopen Brave, if I don't clear my browser data first). This isn't limited to a list of websites either, it is every website that I have tried (from Amazon to my personal websites I'm hosting myself, to many many other websites). I also checked that setting, and allow google logins is already checked. 

Note though that this is not jumping across profiles, as I created a blank test profile with no extensions installed, and the internet/websites are loading as they should (while the original profile open on my other screen still does not load). 

I had created a new profile and connected it to my previous one via sync (so that I could get my extensions added automatically), and the problem persisted. I have noticed however that in my blank profile (not connected to sync, with no extensions installed), the issue does not occur, so it appears to be an issue with an extension. I am going through and adding one extension at a time to see what triggers this behavior, so I may have jumped the gun with this issue.

**Comment by feerlessleadr** [comment]
Ok - so I've figured out the offending extension (although not sure how or what changed, since I've been using this extension for a number of years with no issue). The extension causing the issue is CleanURLs ([here is the link to their site](https://docs.clearurls.xyz/1.23.0/)). It looks like it was updated about the same time that Brave was, and I assumed that it was Brave. I will leave this open for now, but feel free to close, as I will contact the dev of the extension directly. Thanks
